### DISCUSSION TITLE
Custom fields in all areas

### DISCUSSION THREAD

**Discussion by meirinha-lab** [snipe/snipe-it#11361] [Ideas]
I think it would be advantageous to have the possibility to create custom fields in all areas of Snipe-it.

One of the fields should allow the upload of files

**Comment by brianhoganm** [comment]
I agree. Additional fields are needed for people would be great.

**Comment by Delirium-Sama** [comment]
nice ideas consumable and accessory don't use custom field, it could be cool. it could be good to be able to build categories to classified our custom field in the page where we go to build custom field's.

**Reply by meirinha-lab** [reply]
Yes, i agree
